consoles/sshXtermVt.pm

```diff
@@ -41,6 +42,17 @@ sub activate {
     my $sshcommand = $self->sshCommand($username, $hostname, $gui);
     my $serial     = $self->{args}->{serial};
 
+    # Wait that SUT is live on network (for generalhw/ssh)
+    my $p       = Net::Ping->new();
+    my $counter = get_var('SSH_XTERM_WAIT_SUT_ALIVE_TIMEOUT') // 120;
+    while ($counter > 0) {
+        last if ($p->ping($ssh_args->{hostname}));
+        sleep(1);
+        $counter--;
+    }
+    $p->close();
+    bmwqemu::diag("$ssh_args->{hostname} does not seems to be alive. Continue anyway.\n") if ($counter == 0);
```

Comment:
"Continue" sounds like imperative. How about "Continuing"?

Reply:
Or infinitive? ;) I will update it.